packages/app-admin/src/components/views/EnvironmentAliases/graphql.ts

```diff
@@ -0,0 +1,136 @@
+import gql from "graphql-tag";
+
+const fields = `
```

Comment:
These queries are specific to `app-headless-cms`, and are made in the `app-admin` package.

This should not be here.

I suggest you leave this dialog here, but make it also pluggable, so that the additional Headless CMS URLs section can be added from that app, via a plugin.

I see you have the `EnvironmentInfoDialog` component there. I would rename it to `APIInformationDialog`, and add  the Headless CMS section via a new `admin-api-information-dialog` plugin.

Reply:
@EmilK15 this is the same thing you did for the `WelcomeScreen`. Create a skeleton which uses plugins, but the plugins themselves must come from other apps. App dependencies must not be hardcoded inside the packages, meaning, `app-admin` must NOT depend on any other app, because it's a skeleton, not an actual implementation. That's the whole point of plugins.

Reply:
Thanks @Pavel910 for additional explanation.

Reply:
Great points,
So to recap the plan is for that the component within `app-admin` will render a plugin that will be defined within `app-headless-cms` which will contain the environment information content.
In doing this we won't need to make a query that was specific to `app-headless-cms` from `app-admin`.
Makes sense to me

Reply:
Cool, let us know if you'll need any help with this.